docs/content/latest/api/ysql/the-sql-language/statements/cmd_analyze.md

```diff
@@ -0,0 +1,139 @@
+---
+title: ANALYZE statement [YSQL]
+headerTitle: ANALYZE
+linkTitle: ANALYZE
+description: Collect statistics about database tables with the ANALYZE statement.
+beta: /latest/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+menu:
+  latest:
+    identifier: cmd_analyze
+    parent: statements
+aliases:
+  - /latest/api/ysql/commands/cmd_analyze/
```

Comment:
Do we need an alias for this?

Reply:
What alias mean here? I used cmd_copy.md as a template, it has alias.

Reply:
You can delete the aliases section, in that case. (What it does is set up a redirect from any paths you add in the section, all pointing to the current page, but this is only useful if there are links to an existing page, and we need to move that page.)

Reply:
Done.